src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/ProfessionalUserReqValidator.java

```diff
@@ -29,16 +29,14 @@ public static boolean isValidEmail(String email) {
         return false;
     }
 
-    public void validateRequest(String orgId, String showDeleted, String email, String status) {
-        if (null == orgId  && null == email && null == showDeleted) {
+    public void validateRequest(String orgId, String showDeleted, String status) {
```

Comment:
I take some issue with this. It is defensive coding and kind of ugly. If you are expecting nulls, then you need to change the signature of this method to accept an Optional<String> wherever there is a potential null reference being put in.

Please refactor this method signature as discussed above and obviously all of the instances where it is being called.

Reply:
Hi, I did not write this code, I only removed a parameter, but I agree with your comment